fedoracommunity/widgets/package/bugs.py

```diff
@@ -20,15 +20,17 @@ class BugStatsWidget(twc.Widget):
     status_closed_string = "bug_status=CLOSED"
 
     base_query_string = twc.Variable(default='')
-
+    
     def prepare(self):
         super(BugStatsWidget, self).prepare()
-        self.base_query_string = urllib.urlencode({
-            "query_format": "advanced",
-            "product": self.product,
-            "component": self.package,
-        })
-
+        # To get more than one key/value pair with the same key in urllib.urlencode, we need to create a tuple first
+        query_product = (
+            ("query_format", "advanced"),
+            ("product", "Fedora"), 
+            ("product", "FedoraEPEL"),
```

Comment:
```suggestion
            ("product", "Fedora EPEL"),
```

Reply:
Fixed! I thought that bz originally returned `FedoraEPEL`, not `Fedora%20EPEL` in the URL string.